Apollonius_graph_2/include/CGAL/in_place_edge_list.h

```diff
@@ -164,7 +164,7 @@ class In_place_edge_list {
 
   void remove(const Edge& e) {
     CGAL_precondition( is_in_list(e) );
-    static Edge SENTINEL_QUEUE_EDGE = Edge(Face_handle(), -1);
+    static const Edge SENTINEL_QUEUE_EDGE = Edge(Face_handle(), -1);
```

Comment:
`Edge` and `Face_handle` are PODs, as far as I know, so the initialization is actually done at compile time.